app/assets/stylesheets/common/base/login.scss

```diff
@@ -57,10 +57,9 @@
     width: 100%;
     max-width: 100%;
     margin-bottom: 0;
-    min-height: 35px;
-  }
-  #modal-alert:empty {
-    display: none;
+    min-height: 50px;
+    position: absolute;
+    z-index: z("modal", "tooltip");
```

Comment:
I think this absolute positioning here will cause issues: on mobile, on other locales (for example if the message exceeds the vertical space afforded by the 6em padding) or on other modals that don't have the top padding. 

Can you keep the `:empty` pseudoelement, but instead of hiding with `display: none` or `visibility: hidden`, maybe try something like this: 

```
  #modal-alert {
    display: block;
    padding: 1em 3.5em 1em 1em;
    min-height: 35px;
    ...
    &:empty {
      min-height: 0px;
      padding: 0px;
      overflow: hidden;
    }
  }
```

I don't have NVDA handy, but in a regular browser that seems to behave correctly.

Reply:
Sure thanks for the suggestion, I didn't really know how to do it well so will see if this works